Cargo.toml

```diff
@@ -13,17 +13,17 @@ description = "Atomic types of the FuelVM."
 [dependencies]
 rand = { version = "0.8", default-features = false, optional = true }
 serde = { version = "1.0", default-features = false, features = ["derive"], optional = true }
-serde-big-array = { version = "0.3.2", default-features = false, optional = true }
 
 [dev-dependencies]
 bincode = "1.3"
 fuel-types = { path = ".", features = ["random"] }
 hex = "0.4"
 rand = { version = "0.8", default-features = false, features = ["std_rng"] }
+serde_json = "1.0"
 
 [features]
 default = ["std", "serde?/default"]
-alloc = ["serde?/alloc"]
+alloc = ["serde/alloc"]
```

Comment:
```suggestion
```

Reply:
I think deleting `alloc = ["serde/alloc"]` (or more correctly `alloc = ["serde?/alloc"]`) may create a problem at [bytes.rs](https://github.com/FuelLabs/fuel-types/blob/master/src/bytes.rs#L212). 

Should we delete `alloc` from features (and add it as the next suggestion) and then change [bytes.rs](https://github.com/FuelLabs/fuel-types/blob/master/src/bytes.rs#L212) to `#[cfg(feature = "std")]` maybe?